### PR TITLE
feat(style guide): Observability vs Monitoring

### PR DESCRIPTION
**Pull request by austin-schaefer**
Staging draft content from  @bhecknr to clarify usage here, thanks @bhecknr for drafting!

Preview links:

* new `observability vs monitoring` section in https://docswebsitedevelop-observabilityvsmonitoring.gatsbyjs.io/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring
* new `observability` and `monitoring` rows in https://docswebsitedevelop-observabilityvsmonitoring.gatsbyjs.io/docs/style-guide/capitalization/product-capability-feature-usage/#what-not-to-capitalize

### CODE REVIEW

File: src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx [hunk 1]
```diff
@@ -988,17 +1008,17 @@ Notes about features and capabilities:
       </td>
 
       <td>
-        methodology for understanding a complex system
+        the type of platform that New Relic offers (for more information, see [Usage dictionary # Observability vs. monitoring](/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring))
```
Comment (rhetoric101) [inline src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx:1011]:
```suggestion
        the type of platform that New Relic offers (for more information, see [Usage dictionary: Observability vs. monitoring](/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring))
```

**Review by rhetoric101 — DISMISSED** [review]
@austin-schaefer I added a few comments/suggestions. See what you think.

File: src/content/docs/style-guide/word-choice/usage-dictionary.mdx [hunk 2]
```diff
@@ -696,7 +696,102 @@ For either term, in the UI, use the [badge component](https://one-core.newrelic.
...
+    id="observability-monitoring"
+    title="observability vs. monitoring"
+  >
+    Don't use `monitoring` and `observability` interchangeably. Monitoring is a subset of observability. While they're both data-driven, they differ in the questions users can answer with the data they produce.
+
+    Monitoring relies on metrics and produces aggregate measurements. It describes populations and helps users detect broad symptoms and changes in behavior. Has something changed? What direction is it trending in?
+
+    Observability relies on discrete events that carry a lot of detail (context) about the event. It helps users take the questions to the next logical level. What actually changed? What and who was impacted? Where did it come from? Observability helps users characterize behavior at granular levels and narrow down a problem they didn't previously anticipate.
+
+    It’s not an either/or. They're both necessary to troubleshoot a problem quickly. New Relic is an observability platform that includes multiple types of monitoring capabilities, such as infrastructure monitoring, application performance monitoring, network monitoring, and so on.
```
Comment (paperclypse) [inline src/content/docs/style-guide/word-choice/usage-dictionary.mdx:708]:
```suggestion
Don’t use `monitoring` and `observability` interchangeably.

We describe New Relic as an observability platform that includes multiple types of monitoring tools. You can see this in many of our capability names: application performance monitoring, infrastructure monitoring, network monitoring, and so on.

In general, use observability to describe the New Relic platform and the health of a system holistically based on different types of data. Use monitoring when you’re describing specific capabilities and the data from those capabilities.

This is a guideline, not a rule. The most important thing is that you use these terms thoughtfully.
```

**Review by paperclypse — DISMISSED** [review]

File: src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx [hunk 3]
```diff
@@ -1044,6 +1044,26 @@ Notes about features and capabilities:
...
+      <td>
+        monitoring
+      </td>
+
+      <td>
+        a subset of observability that describes specific capabilities of the New Relic observability platform, such as infrastructure monitoring or application performance monitoring (for more information, see [Usage dictionary:  Observability vs. monitoring](/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring))
+      </td>
+
+      <td>
+        first use: monitoring
```
Comment (paperclypse) [inline src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx:1057]:
```suggestion
        First use: monitoring
```

File: src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx [hunk 4]
```diff
@@ -1025,24 +1025,25 @@ Notes about features and capabilities:
...
 
-<tr>
+    <tr>
       <td>
         monitoring
       </td>
 
       <td>
-        In general, use monitoring when you’re describing specific capabilities and the data from those capabilities. Monitoring is watching something over time to detect broad symptoms and changes. See observability entry to compare differences.
+        Use monitoring when you're describing specific capabilities and the data from those capabilities. (for more information, see [Usage dictionary:  Observability vs. monitoring](/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring))
```
Comment (paperclypse) [inline src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx:1034]:
```suggestion
        Use monitoring when you're describing specific capabilities and the data from those capabilities. (for more information, see [Usage dictionary: Observability vs. monitoring](/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring))
```

File: src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx [hunk 5]
```diff
@@ -1068,7 +1069,7 @@ Notes about features and capabilities:
       </td>
 
       <td>
-        In general, use observability to describe the New Relic platform and the health of a system holistically based on different types of data. Observability is using the outputs of a system, including different types of data, to understand its health, enabling engineers to gain insights and diagnose issues effectively. See monitoring entry to compare differences.
+        In general, use observability to describe the New Relic platform and the health of a system holistically based on different types of data. Observability is using the outputs of a system, including different types of data, to understand its health, enabling engineers to gain insights and diagnose issues effectively. (for more information, see [Usage dictionary:  Observability vs. monitoring](/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring))
```
Comment (paperclypse) [inline src/content/docs/style-guide/capitalization/product-capability-feature-usage.mdx:1072]:
```suggestion
        In general, use observability to describe the New Relic platform and the health of a system holistically based on different types of data. Observability is using the outputs of a system, including different types of data, to understand its health, enabling engineers to gain insights and diagnose issues effectively. (for more information, see [Usage dictionary: Observability vs. monitoring](/docs/style-guide/word-choice/usage-dictionary/#observability-monitoring))
```

**Review by paperclypse — COMMENTED** [review]
Fixing some minor typos. For some reason if there are two spaces after a colon, it seems to convert it into a hash

**Review by paperclypse — APPROVED** [review]
OK, I've finally gotten this to a place where I'm happy with it anyway

**Pull request merged**